presto-main/src/test/java/com/facebook/presto/execution/TestCommitTask.java

```diff
@@ -94,7 +94,8 @@ public void testNoTransactionCommit()
                 fail();
             }
             catch (CompletionException e) {
-                throw Throwables.propagate(e.getCause());
+                throwIfUnchecked(e.getCause());
```

Comment:
I think the comment about unwrapping an error from the other Thread also applies here

Reply:
Actually, this is legacy from `CompletableFuture` and is not needed anymore. I'll remove it.

Reply:
Ok, that was in the test. I missed that.